src/styles/_catalog.scss

```diff
@@ -1,22 +1,38 @@
+$cardWidth: 200px;
+$cardHeight: 300px;
+$cardGap: 48px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  column-gap: $cardGap;
+  row-gap: $cardGap;
+  padding-bottom: 48px;
 
-  max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    max-width: calc(#{$cardWidth} * 2 + #{$cardGap});
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    max-width: calc(#{$cardWidth} * 3 + #{$cardGap} * 2);
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    max-width: calc(#{$cardWidth} * 2 + #{$cardGap} * 3);
+    grid-template-columns: repeat(4, 1fr);
+  }
+
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    box-sizing: border-box;
```

Comment:
it's extra bc you already have this in main.scss
![image](https://user-images.githubusercontent.com/91411692/222469737-8703c1ee-e815-46dd-b1a5-d4ae83c35c1a.png)

```suggestion
```